packages/circuit-ui/components/ListItem/ListItem.tsx

```diff
@@ -65,16 +64,16 @@ interface BaseProps {
    * Display a trailing label.
    * If using the `navigation` variant, the chevron icon will be center aligned with this label.
    */
-  suffixLabel?: ReactNode;
+  trailingLabel?: string | ReactNode;
```

Comment:
The `trailingLabel` and `trailingDetails` are strings, but their text formatting can vary.

I'd love to have made the type `string | BodyProps`, but the `Body` component doesn't support strikethrough text, and this is one of the explicit example uses of the component (for example, when rendering failed transactions).

Instead, I suggest making it `string | ReactNode` to hint at the fact that this can be a simple string for simplicity, or a custom ReactNode for other use cases. This is also in line with implementation, which renders a default `Body` element if `typeof p === "string"`

Reply:
Actually, now that I've written all of this, I'm realizing that if we allow `BodyProps`, developers might be able to pass a `css` prop with strikethrough styles:

```jsx
<ListItem
  label: "Failed transaction"
  trailingLabel: {
    children: "12 €",
    css: { textDecoration: "line-through" }
  }
/>
```

...but we might have to explicitly add `css` to the `BodyProps` for TS to be happy with this.

The benefit is that it would be clearer that the purpose of these `trailingLabel` and `trailingDetails` props is to render text, and that custom use cases should use `trailingElement` instead

Reply:
Update: I think it's best to keep the current interface here to support other custom `Body`-like cases, like [multi-line labels](https://circuit.sumup.com/?path=/story/components-listitem--with-custom-label).